### PR TITLE
Change servo to uppercase and links to Servo and Rust

### PR DESCRIPTION
**Pull request by Coder206**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,12 +40,12 @@
             <p>
             Sometimes it's hard to know where to get started, though.
             <em>Servo Starters</em> is a list of easy tasks that are good for
-            beginners to rust or servo.
+            beginners to <a href="https://github.com/rust-lang/rust">Rust</a> or <a href="https://github.com/servo/servo">Servo</a>.
```
Comment (jdm) [inline index.html]: I think linking to https://rust-lang.org/ would make more sense here.

Comment (Coder206) [reply index.html]: @jdm and https://servo.org?

Comment (jdm) [reply index.html]: Sure.

**Comment by jdm** [comment]
Closed in favour of #52.

**Pull request closed without merging**